tools/docker-compose/ansible/roles/sources/templates/docker-compose.yml.j2

```diff
@@ -60,6 +60,7 @@ services:
 {% if cluster_node_count|int > 1 %}
   haproxy:
     image: haproxy
+    user: root
```

Comment:
```suggestion
```

Reply:
`user: "1000"` works

not specifying a user leads to an error, not sure exactly why